G/gmsh/build_tarballs.jl

```diff
@@ -30,22 +38,48 @@ sed ${prefix}/lib/gmsh.jl.bak \
   -e 's/^\(const lib.*\)/#\1/g' \
   -e 's/^\(module gmsh\)$/\1\nusing gmsh_jll: libgmsh\nconst lib = libgmsh/g' \
   > ${prefix}/lib/gmsh.jl
+install_license ../LICENSE.txt
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = expand_cxxstring_abis(supported_platforms())
+filter!(p -> !(Sys.iswindows(p) && cxxstring_abi(p) == "cxx03"), platforms) # Restriction from OCCT builds
 
 # The products that we will ensure are always built
 products = [
     LibraryProduct(["libgmsh", "gmsh"], :libgmsh),
-    FileProduct("lib/gmsh.jl",:gmsh_api)
+    FileProduct("lib/gmsh.jl", :gmsh_api)
 ]
 
+# Some dependencies are needed only on Linux and FreeBSD
+x11_platforms = filter(p ->Sys.islinux(p) || Sys.isfreebsd(p), platforms)
+
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
+    BuildDependency("Xorg_xorgproto_jll"; platforms=x11_platforms),
+    Dependency("Cairo_jll"),
+    Dependency("CompilerSupportLibraries_jll"),
```

Comment:
Is this needed only for libgomp?

Reply:
I believe so, this was the only dependency for the prior 4.9.3 build. It seems like it finds OpenMP and builds fine without it, though.

Reply:
Then I wanted to suggest to limit it to `!Sys.isbsd` platforms (for the others we have llvm openmp), not to remove it completely 🙂

Reply:
🙂